core/components/footer/fragment.ts

```diff
@@ -26,7 +26,7 @@ export const FooterFragment = graphql(`
       }
     }
     content {
-      pages(filters: { isVisibleInNavigation: true }) {
+      pages(filters: { parentEntityIds: [0] }) {
```

Comment:
What does this change? Only fetch the root pages?

Reply:
@jorgemoya Yes, exactly. For the footer, we only want the top-level pages, and also we want to include web pages that don't have `isVisibleInNavigation: true` - this value is only pertinent to what is displayed in the nav bar, not the footer.